vsintegration/src/FSharp.Editor/QuickInfo/QuickInfoProvider.fs

```diff
@@ -218,7 +218,10 @@ type internal FSharpQuickInfoProvider
                         |> Seq.map (fun x -> x.Text.Length)
                         |> Seq.max
 
-                    let seperator = TaggedTextOps.tag Text (String.replicate width "-")  
+                    // eyeballed formula to prevent the divider from wrapping
+                    let width = if width / 2 > 85 then 85 else width / 2
```

Comment:
What are the units of 2 and 85?  Would perhaps a unit of measure defined for these values help in letting people understand what the numbers are for?